watson_developer_cloud/websocket/speech_to_text_websocket_listener.py

```diff
@@ -0,0 +1,174 @@
+            self.sendMessage(json.dumps(init_data).encode('utf8'))
+
+            # start sending audio right away (it will get buffered in the STT service)
+            self.send_audio(self.audio.read())
+            log.msg("onOpen ends")
+
+        def onMessage(self, payload, isBinary):
+            json_object = json.loads(payload.decode('utf8'))
+
+            if 'error' in json_object:
+                log.msg('Error : {}'.format(json_object['error']))
+                # Only call on_error() if a real error occurred. The STT service sends
+                # // {"error" : "No speech detected for 5s"} for valid timeouts, configured by
+                # options.inactivity_timeout
+                error = json_object['error']
+                if error.startswith(self.TIMEOUT_PREFIX):
+                    self.callback.on_inactivity_timeout()
+                else:
+                    self.callback.on_error()
+
+            # if uninitialized, receive the initialization response from the server
+            elif 'state' in json_object:
+                self.listening_messages += 1
+                if self.listening_messages == 1:
+                    self.callback.on_listening()
+
+                elif self.listening_messages == 2:
+                    # close the connection
+                    self.sendMessage(self.build_close_message())
+                    self.callback.on_transcription_complete()
+                    self.sendClose(1000)
+
+            # if in streaming
+            elif 'results' in json_object or 'speaker_labels' in json_object:
+                hypothesis = ""
+                # empty hypothesis
+                if len(json_object['results']) == 0:
+                    print("empty hypothesis!")
+                # regular hypothesis
+                else:
+                    hypothesis = json_object['results'][0]['alternatives'][0][
+                        'transcript']
+                    b_final = (json_object['results'][0]['final'] == True)
+                    transcripts = self.extract_transcripts(
+                        json_object['results'][0]['alternatives'])
+
+                    if b_final:
+                        self.callback.on_hypothesis(hypothesis)
+                    else:
+                        self.callback.on_transcription(transcripts)
+
+        def onClose(self, wasClean, code, reason):
+            log.msg("onClose, WebSocket connection closed: {0}".format(reason),
+                    "code: ", code, "clean: ", wasClean, "reason: ", reason)
+            self.factory.endReactor()
+
+    class WSInterfaceFactory(WebSocketClientFactory):
+        def __init__(self, audio, options, callback, url=None, headers=None):
+            WebSocketClientFactory.__init__(self, url=url, headers=headers)
+            self.audio = audio
+            self.options = options
+            self.callback = callback
+
+            self.openHandshakeTimeout = 6
```

Comment:
What's 6? is it seconds? milliseconds?.
Please use a constant like `SIX_SECONDS`